rotkehlchen/history/skipped.py

```diff
@@ -109,11 +110,15 @@ def reprocess_skipped_external_events(rotki: 'Rotkehlchen') -> tuple[int, int]:
 
         total_num += len(raw_events)
         exchange = cast('Kraken', exchange)
-        _, processed_refids = exchange.process_kraken_raw_events(
+        new_events, processed_refids = exchange.process_kraken_raw_events(
             events=[x[1] for x in raw_events],
             events_source='processing skipped events',
             save_skipped_events=False,
         )
+        if len(new_events) != 0:
```

Comment:
Discussed with Yabir that this logic is only in Kraken. It's ugly and probably does not scale. We have this old issue to implement in other exchanges but I don't see it happening: https://github.com/rotki/rotki/issues/3635


Not an action item for this PR, but perhaps we should try to see what would it mean to remove them.